test/jobs/github-event/pull_request/opened.js

```diff
@@ -53,11 +68,48 @@ test('github-event pull_request opened', async t => {
     t.end()
   })
 
+  t.test('initial pr on private repo opened', async t => {
```

Comment:
❓ why did you add this test?

Reply:
Same as https://github.com/greenkeeperio/greenkeeper/pull/602#discussion_r164702375